bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBridgeHandler.java

```diff
@@ -303,16 +300,16 @@ public void responseReceived(RDVersionResponse response) {
 
     @Override
     public synchronized void dispose() {
-        EnOceanTransceiver localTransceiver = transceiver;
-        if (localTransceiver != null) {
-            localTransceiver.shutDown();
-            transceiver = null;
+        EnOceanTransceiver transceiver = this.transceiver;
```

Comment:
Why revert this? I think it is perfectly fine.

Reply:
See other comments.